Makefile

```diff
@@ -0,0 +1,24 @@
+.DEFAULT_GOAL := help
```

Comment:
I would like to avoid adding the Makefile as this is only a library and the usage of the tools in commands is pretty standard. Thanks for writing it, but I think that it is not adding much.

Reply:
the use case i saw is to improve developer speed. `make lint` as opposed to waiting for a github action to run is a lot faster

Reply:
I would prefere not to add the Makefile for a library as these commands are pretty standard, there is nothing so specific to require the complexity of makefile and the additional file in the repo. If you would like to remove it, the rest of the pr is good.